include/devicetree/pm_state.h

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2020 Intel corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DEVICETREE_PM_STATE_H_
+#define ZEPHYR_INCLUDE_DEVICETREE_PM_STATE_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <power/power_state.h>
```

Comment:
Devicetree headers shouldn't need to rely on higher level system headers, so this looks like a layering inversion to me. This include should be removed. In general we need to be able to pull in devicetree.h from linker scripts, and we need all the private devicetree header files to be included in devicetree.h, so this include should be done somewhere else, by the C files that need this.